### ISSUE TITLE
Building with Carthage: No such module 'FBSDKShareKit'

### ISSUE THREAD

**Issue by nicolas-miari** [facebook/facebook-swift-sdk#500]
## Checklist

- [✔︎] I've updated to the latest released version of the SDK
- [✔︎ ] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [✔︎] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [✔︎] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [✔︎] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `10.2`
- Swift Version: `5.0`
- Installation Platform & Version: `[Carthage]` version `X.X.X`

## Goals

Build my project using Fastlane on Travis.

## Expected Results

Build succeeds.

## Actual Results

Build fails with:

```
 (path to source file): no such module 'FBSDKShareKit'

 import FBSDKShareKit
        ^
```

## Steps to Reproduce

Build app project using the following syntax in Cartfile:
 
    github "facebook/facebook-swift-sdk" "master"

## Code Samples & Details

```swift
import FBSDKShareKit
```

**Comment by nicolas-miari** [comment]
This started happening today. We have been running automated builds recently, a few days ago at least. We haven't made any changes to our source code that could possibly break this...

**Comment by nicolas-miari** [comment]
I really don't understant what's going on. 

The dependency facebook-objc-sdk (which includes the FBSDK* frameworks) is being checked out, but for some reason it'ws not being built? My app project suddenly cannot find the framework after cleaning the carthage dependencies.

If I instead use:

    import FacebookShare

I get the compile error:

> Missing required module 'FBSDKShareKit'

**Comment by nicolas-miari** [comment]
I got carthage to build the Objective-C frameworks by adding the `--no-use-binaries` switch, and now my project compiles.

I don't fully understandd what has changed, that I can no longer get away with binaries.

**Comment by nicolas-miari** [comment]
I tried pinning **both** SDKs (Swift and Objective-C) to their previously resolved version/commit, but Carthage still does not build the objc SDKs.

**Comment by JosephDuffy** [comment]
I think this was caused by https://github.com/facebook/facebook-objc-sdk/commit/86138e2db03726f580a564f389dc81fb88cd81e5. I think Carthage downloads the first `.zip` it finds attached to a release, so it's now downloading the AccountKit.framework.zip file, which only contains AccountKit.

From a Cartahge point of view I don't think there's a need for the other archives, FacebookSDK_Dynamic.framework.zip contains all them. Maybe there's a way to name the archives in a way that causes Carthage to get the right one.

**Comment by dreamolight** [comment]
facebok-objc-sdk release has been updated, now carthage could pull all the kits correctly, which fixes this issue.

**Comment by dreamolight** [comment]
Renaming released SDK will cause another issue, I'll release the objc SDK to fix this issue this weekend. So reopen it for now as it's not been fixed.

**Comment by dreamolight** [comment]
We released a new version of objc SDK: 5.4.0 which fixed this issue.
